### DISCUSSION TITLE
Filters hided with a warning

### DISCUSSION THREAD

**Discussion by Andrewblasco** [tuskyapp/Tusky#3002] [Ideas]
In Mastodon web, there are a section in Settings called Filters.
There, any user could filter certain words to prevent to appear into timeline.
In each filter, you have the option to hide completely these words but ALSO the option to hide those words with a warning.

I've noticed that Tusky can't show that warning, just hide all the filters, disregarding the ajustments.
I think is a useful feature and it would be nice to have completely implemented on Tusky.

**Comment by nikclayton** [comment]
This was implemented in Tusky 22.0 -- thanks.
